contents/blog/posthog-cloud-eu.md

```diff
@@ -0,0 +1,54 @@
+---
+date: 2022-10-10
+title: Introducing PostHog Cloud EU
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ["Product updates”]
+author: ["andy-vandervell"]
+featuredImage: ../images/blog/posthog-eu-blog.png
+featuredImageType: full
+--
+
+Today, we’re delighted to announce the [launch of PostHog Cloud EU](/eu) – a fully-managed version of PostHog where all data is hosted exclusively in the EU.
```

Comment:
'Delighted to announce' is pretty generic and not opinionated.

I'd propose cutting this first line and jumping straight to the real point. 
```suggestion
It's no secret that some popular analytics tools, such as [Google Analytics](https://isgoogleanalyticsillegal.com/), have fallen foul of GDPR regulations in the EU. That's one reason why we've enabled users to self-host PostHog, or to anonymize user data in a way which complies with GDPR. 

Today, we're making a third option available: hosting PostHog Cloud in Europe so that user data never has to leave the EU. Now, any business which needs to comply with GDPR regulations can get up and running in a few minutes, and without the ongoing upkeep required for a self-hosted deployment. 

Oh, and PostHog Cloud EU also happens to be faster for users located in Europe, which is nice. 

PostHog Cloud EU offers everything you'd expect of PostHog - analytics, session recording, experiments and more. The only difference is that you can now choose to host in a different region, at no extra cost. 
```